include/sys/atomic.h

```diff
@@ -142,14 +142,15 @@ static inline bool atomic_test_bit(const atomic_t *target, int bit)
  *
  * @return true if the bit was set, false if it wasn't.
  */
-static inline bool atomic_test_and_clear_bit(atomic_t *target, int bit)
+static inline bool atomic_test_and_clear_bit(atomic_t *target, unsigned int bit)
 {
-	atomic_val_t mask = ATOMIC_MASK(bit);
+	unsigned int mask = ATOMIC_MASK(bit);
```

Comment:
There's a risk of introducing a bug if/when this gets forward-ported to main. There the underlying type of `atomic_val_t` has been changed from `int` to `long`, so it's critical that the width of the mask and the value it's applied to have the same width. @cfriedt FYI

Reply:
Is it ok to introduce uatomic_val_t for an unsigned variant?

Reply:
I think it'd indeed be good to have a typedef that's compatible with the width of `atomic_val_t`, however we should retain the existing namespace, i.e. `atomic_*` instead of coming up with a new one. How about `atomic_bits_t`?

Reply:
I like it very much.

Reply:
Works for me. Note @Abramo-Bagnara - the eventual goal is to switch to using the C11 atomic API, just in case that needs to go into some agenda.